docs/2023/Sunbird inQuiry/Auto generates questions in QuML/updates/2023-07-07.md

```diff
@@ -4,13 +4,26 @@ author: Shlok
 ---
 
 ## Milestones
-- [ ] Give the description about Milestone 1
-- [ ] Give the description about Milestone 2
-- [ ] Give the description about Milestone 3
-- [ ] Give the description about Milestone 4
+- [x] Model Exploration for Question Generation
+- [x] Selecting the T5 Model for Question Generation
+- [x] Text Extraction from PDF using PyPDF2
 
 ## Screenshots / Videos 
+PDF Reader
+<br></br>
+<img width="386" alt="Screenshot 2023-07-21 at 11 29 19 PM" src="https://github.com/ShlokRana/c4gt-milestones/assets/62721720/93b80486-dc9b-421e-ad23-328d18199a63">
+<br></br>
 
-## Contributions
+Extracted text from PDF
+<br></br>
+<img width="692" alt="Screenshot 2023-07-21 at 11 30 12 PM" src="https://github.com/ShlokRana/c4gt-milestones/assets/62721720/0a740422-83cd-4d05-b6b0-cddbfda3ba58">
```

Comment:
Do not add screenshots here. Please add them to screenshots and video section. In this action please add the issue you you are resolving or the PR raised on your project.